src/monali/TestCase_12_1.java

```diff
@@ -0,0 +1,35 @@
+package monali;
+
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+//program 1: emplyees count in each deptName.
+
+
+public class TestCase_12_1 {
+	
+	static HashMap<String,Integer> getEmpCount(WebDriver driver){ 
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method